components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -229,6 +230,47 @@ subroutine ocn_diagnostic_solve(dt, statePool, forcingPool, meshPool, scratchPoo
              vertGMBolusVelocityTop, relativeVorticityCell, divergence, &
              kineticEnergyCell, tangentialVelocity)
 
+      if ( config_vert_advection_method == 'remap' ) then
+
+         ! Overwrite vertVelocityTop with exact Eulerian solution from remapping
+         call mpas_pool_get_array(statePool, 'layerThicknessLag', &
+             layerThicknessLag, timeLevel)
+         call ocn_diagnostic_solve_vertVel_remap(layerThickness, &
+             layerThicknessLag, dt, vertVelocityTop)
+
+         ! Overwrite vertical tracer budget terms with new vertVelocityTop
```

Comment:
As far as I can tell, there are only budgets for active tracers. If you're aware of other relevant tracer variables, can you point me to them?

Reply:
Apologies.  I would have assumed this would have been done for all tracers rather than just for active tracers but I see that wasn't the case: https://github.com/MPAS-Dev/MPAS/pull/1381